docs/ota-client-guide/modules/ROOT/pages/virtual-secondaries.adoc

```diff
@@ -0,0 +1,55 @@
+= Use of Virtual Secondaries
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+
+Virtual secondary devices can be used as a way to update a configuration file on the primary device (where the primary device is a standard OSTree image running an aktualizr). You can do this in two parts:

```

Comment:
"Device" is a word we have to be careful about, since usually we use the word to refer to an entire system, e.g. a car. I think we're trying to move away from referring to individual units as "ECUs" since that is quite automotive-specific, but I'm not sure what a better term is. You might be able to just say "the primary".